### PR TITLE
Adds GitLab CI Example

### PR DESCRIPTION
**Pull request by aknosis**
Adds an extremely basic example configuration. There are many many variations one may need depending on the specific project but this covers the basics of install php dependencies and deploying with serverless.

Also going to add a note specific to deploying Laravel.

### CODE REVIEW

File: CI/GitLab/.gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    - vendor/
+
+build:
+  stage: build
+  image: php:7.4
+  script:
+    - apt-get update
+    - apt-get install unzip
+    - curl -sS https://getcomposer.org/installer | php
+    - php composer.phar install --no-dev --no-progress --prefer-dist
```
Comment (mnapoli) [inline CI/GitLab/.gitlab-ci.yml:16]: Why not use the [`composer` Docker image](https://hub.docker.com/_/composer)? That way no need to update/install everything.

File: CI/GitLab/.gitlab-ci.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+deploy:
+  stage: deploy
+  image: node:alpine
+  script:
+    - npm install -g serverless
+    - serverless deploy -v -s staging
```
Comment (mnapoli) [inline CI/GitLab/.gitlab-ci.yml:23]:
```suggestion
    - serverless deploy -v
```

Let's remove the stage option in order to keep the example as simple (to the point) as possible. Deploying with a specific stage is not required.

**Review by mnapoli — COMMENTED** [review]
Looking good! I added 2 inline suggestions.

File: CI/GitLab/.gitlab-ci.yml (hunk 2 above)
Comment (aknosis) [reply CI/GitLab/.gitlab-ci.yml:23]: Makes sense.

File: CI/GitLab/.gitlab-ci.yml (hunk 1 above)
Comment (aknosis) [reply CI/GitLab/.gitlab-ci.yml:16]: The issue with the composer image is the inability to choose your php version. With this method you can choose any php version. This is something I've had to do with my projects and so I carried over the practice in this example. 

However, in the interest of brevity I will change to the composer image.

**Review by mnapoli — APPROVED** [review]
Thank you!

**Pull request merged**